src/ConcordiumNetSdk/Types/MicroCcdAmount.cs

```diff
@@ -0,0 +1,142 @@
+﻿namespace ConcordiumNetSdk.Types;
+
+/// <summary>
+/// Represents an amount in µCCD.
+/// Note that 1_000_000 µCCD is equal to 1 CCD.
+/// </summary>
+public readonly struct MicroCCDAmount : IEquatable<MicroCCDAmount>
```

Comment:
We just call this Amount elsewhere.

"MicroCCD" is one way to view it and construct it.